components/plugin_marketplace/marketplace_list/marketplace_list.tsx

```diff
@@ -4,46 +4,56 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+import {MarketplacePlugin} from 'mattermost-redux/types/plugins';
+
 import MarketplaceItem from '../marketplace_item';
 
 import NavigationRow from './navigation_row';
 
 const PLUGINS_PER_PAGE = 15;
 
-export default class MarketplaceList extends React.PureComponent {
+type MarketplaceListProps = {
+    plugins: MarketplacePlugin[];
+};
+
+type MarketplaceListState = {
+    page: number;
+};
+
+export default class MarketplaceList extends React.PureComponent <MarketplaceListProps, MarketplaceListState> {
     static propTypes = {
```

Comment:
propTypes is not needed anymore, since the props are already typed.

Reply:
Good catch, fixed in 125dd8947